packages/uswds/src/Templates/FieldTemplate.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import {
+  FieldTemplateProps,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+  getTemplate,
+  getUiOptions,
+} from "@rjsf/utils";
+
+/** The `FieldTemplate` component is the template used by `SchemaField` to render any field. It renders the field
+ * content, (label, description, children, errors and help) inside of a `div` with the `usa-form-group` class
+ * name applied.
+ *
+ * @param props - The `FieldTemplateProps` for this component
+ */
+const FieldTemplate = <
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>(
+  props: FieldTemplateProps<T, S, F>,
+) => {
```

Comment:
We prefer functions to fat-arrows
```suggestion
export default function FieldTemplate = <
  T = any,
  S extends StrictRJSFSchema = RJSFSchema,
  F extends FormContextType = any,
>(
  props: FieldTemplateProps<T, S, F>,
) {
```